### ISSUE TITLE
invalid_client connecting to soundcloud

### ISSUE THREAD

**Issue by EugeneFeingold** [streamcloudapp/StreamCloud#101]
This might well be on SoundCloud's end, but I get this error trying to connect to soundcloud:

url: https://soundcloud.com/connect?client_id=909c2edcdbd7b312b48a04a3f1e6b40c&response_type=code&redirect_uri=streamcloud%3A%2F%2Fsoundcloud%2Fcallback

response: invalid_client: The client identifier provided is invalid.

**Comment by Kraahkan** [comment]
Can confirm! 

**Screencap** http://jmp.sh/K9nnkcu

**Comment by plaetzchen** [comment]
Yes, this is an issue. SoundCloud revoked our API key and I am not completely sure why. I will try to contact someone there and see what's the issue.

**Comment by lngsx** [comment]
confirm

**Comment by theapache64** [comment]
Issue still exists.

**Comment by plaetzchen** [comment]
So SoundCloud finally decided that our app is "to close to the original product" for what ever that means. The only way to fix it is that you create your own API key and use that one for your build.

**Comment by Kraahkan** [comment]
Thanks for the update. That is bewildering to hear

**Comment by 0xYasser** [comment]
They closed the ability to register new apps too. so we cannot get our own keys. rip

**Comment by theapache64** [comment]
What if i donate a working soundcloud API key ?
